src/components/view/ConnectDevicesView/ConnectDevicesView.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { BlankView, Card } from "../.."
+import { ConnectDevicesMenu, FitbitDevices, GarminDevices, ViewDeviceActivity } from '../../container'
+import MyDataHelps from '@careevolution/mydatahelps-js'
+import standardDailyDataTypes from '../../../helpers/daily-data-types/standard'
+
+export interface ConnectDevicesViewProps {
+    externalAccountsApplicationUrl: string
+    presentation?: "Modal" | "Push"
+    previewState?: "default"
+    colorScheme?: "auto" | "light" | "dark"
+    enableAppleHealthSurveyName?: string
+    enableGoogleFitSurveyName?: string
+    deviceActivityViewUrl?: string
+}
+
+export default function ConnectDevicesView(props: ConnectDevicesViewProps) {
+    return (
+        <BlankView showBackButton={props.presentation == "Push"}
+            showCloseButton={props.presentation == "Modal"}>
+            {props.deviceActivityViewUrl &&
+                <Card>
+                    <ViewDeviceActivity onClick={() => MyDataHelps.openApplication(props.deviceActivityViewUrl!)}
+                        dataTypes={standardDailyDataTypes}
+                        previewState={props.previewState ? "fetchingData" : undefined} />
+                </Card>
+            }
+            <Card>
+                <ConnectDevicesMenu previewState={props.previewState ? "Web" : undefined}
+                    enableAppleHealthSurveyName={props.enableAppleHealthSurveyName}
```

Comment:
This may be WIP but noting in case not. This does not match the parameter name.

Reply:
I'm also seeing the Build and test workflow did not pick this up. I will work on adding problems things like this to Build and test workflow

Reply:
I did rename this parameter and I think it's consistent / working?

Reply:
Yes, I did see the correction